### DISCUSSION TITLE
Open charger on door Ioniq 5

### DISCUSSION THREAD

**Discussion by nixxxor** [Hyundai-Kia-Connect/kia_uvo#303] [Ideas]
Hello, would it be possible to add functionality to open the charger door on a Ioniq 5?

I want to be able to do this to make an automation where the charger door opens automatically when I lift the handle from the charging station at home.

Is there anything I can provide to help acomplish this (I'm not a developer)?

**Comment by cdnninja** [comment]
This is being tracked here: https://github.com/fuatakgun/kia_uvo/issues/243 it will probably be a little bit until we get to it but if it can be done in the app we could do it here.
